LessonDBBottomNavSwipe/app/src/main/java/dev/software/db/adapter/ListAdapter.kt

```diff
@@ -0,0 +1,78 @@
+package dev.software.db.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import dev.software.db.databinding.ItemBinding
+import dev.software.db.model.Song
+
+class ListAdapter(
+        private val onClicked: (item: Song) -> Unit
+) : ListAdapter<Song, RecyclerView.ViewHolder>(DIFF_UTIL) {
```

Comment:
у тебя 1 тип ViewHolder
```suggestion
) : ListAdapter<Song, SongViewHolder>(DIFF_UTIL) {
```